tuning/config/peft.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass, field
+from typing import List
+
+
+@dataclass
+class lora_config:
+    r: int = 16
```

Comment:
why such a large default value for `r`?

llama-recipes uses 8. The original paper doesn't test 16 (when they explore "optimal rank r" they test 1,2,4,8, and 64) concluding "LoRA already performs competitively with a very small r"

Reply:
@raghukiran1224 Any reason using 16?  I think I might copied this value from your hardcoded version.

Reply:
No reason, we can change defaults once we know what works best.

Reply:
will make it 8 then as that seems to be a more popular default value.